index.js

```diff
@@ -0,0 +1,36 @@
+const $taskInput = document.getElementById('taskInput');
+const $taskList = document.getElementById('taskList');
+
+const handleKeydownTaskInput = (event) => {
+  const taskText = $taskInput.value.trim();
+
+  if (event.key !== 'Enter' || taskText === '') {
+    return;
+  }
+
+  const taskItem = createTaskItemElement(taskText);
+  $taskList.appendChild(taskItem);
+  $taskInput.value = '';
+};
+
+$taskInput.addEventListener('keydown', handleKeydownTaskInput);
+
+const createTaskItemElement = (text) => {
+  const taskItemLi = document.createElement('li');
+  const taskTextSpan = document.createElement('span');
+  const deleteButtonImg = document.createElement('img');
+  var svgImage =
```

Comment:
코드를 긁어와서 사용해서 var로 변수가 선언되어 있는 것 같아요~
var는 현대 개발에서는 거의 금기시해야 합니다!

```suggestion
  const svgImage =
```

Reply:
넵 유의하겠습니다!